src/openms/source/FORMAT/MSPGenericFile.cpp

```diff
@@ -126,21 +127,25 @@ namespace OpenMS
         spectrum.setName( String(m[1]) );
         spectrum.setMetaValue("is_valid", 1);
       }
+      // Specific case of NIST's exported msp
```

Comment:
I wonder how many other "specific" cases there are for the other databases.  This maybe a detail worth documenting in the docstrings for future developers (or even us) if we have to add in another "specific" case at some point in time.

Reply:
Yeah, the absence of precise rules is not cool.

I explain the problem in better words in the following [FIX] PR to upstream: https://github.com/OpenMS/OpenMS/pull/3574